qiskit/providers/ibmq/experiment/experimentservice.py

```diff
@@ -383,6 +386,8 @@ def upload_analysis_result(self, result: AnalysisResult) -> None:
             data['tags'] = result.tags
         if result.uuid:
             data['uuid'] = result.uuid
+        if result.verified:
```

Comment:
```suggestion
        if result.verified is not None:
```

Same thing as above, booleans are tricky.